handle/abandon.go

```diff
@@ -0,0 +1,23 @@
+package handle
+
+import (
+	"context"
+
+	servicebus "github.com/Azure/azure-service-bus-go"
+)
+
+// Abandon stops processing the message and releases the lock on it.
+func Abandon() Handler {
+	return &abandon{}
+}
+
+type abandon struct {
+}
+
+func (a *abandon) Do(ctx context.Context, _ Handler, message *servicebus.Message) Handler {
+	if err := message.Abandon(ctx); err != nil {
+		// trace the failure to abandon the message.
+		// the processing will terminate and the lock on the message will be released after messageLockDuration
```

Comment:
in essence an error just makes us wait longer.

Reply:
Probably going to integrate opengelemetry/opencensus.
So if that's your tracing framework, it removes the code than needs to write the span from your business logic.